slices/git.yaml

```diff
@@ -0,0 +1,41 @@
+package: git
+
+essential:
+  - git_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/git:
+      /usr/bin/git-shell:
+      /usr/bin/scalar:
+
+  core:
+    essential:
+      - libc6_libs
+      - libcurl3t64-gnutls_libs
+      - libexpat1_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+      - git_bins
```

Comment:
You could omit `libc6`, `libpcre2..` and `zlib1g` items from the list since `git_bins` already has them.

Reply:
Fixed! Good catch